File: Week6/sentimental-mario-more/mario.py
from cs50 import get_int

while True:
    n = get_int("Height: ")
    if n > 0 and n <= 8:
        break

for a in range(n):

    print(" " * (n - a - 1), end="")
    print("#" * (a + 1), end="")
    print("  ", end="")
    print("#" * (a + 1))

File: Week1/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);

    for (int a = 0; a < height; a++)
    {
        for (int b = 0; b < height + a + 3; b++)
        {
            if (b == height || b == height + 1 || a + b < height - 1)
            {
                printf(" ");
            }
            else
            {
                printf("#");
            }
        }
        printf("\n");
    }
}
